frontend/src/lib/k8s/cluster.ts

```diff
@@ -732,13 +768,14 @@ export function makeKubeObject<T extends KubeObjectInterface | KubeEvent>(
 
     patch(body: OpPatch[]) {
       const patchMethod = this._class().apiEndpoint.patch;
-      const args: Parameters<typeof patchMethod> = [body];
+      const args: Partial<Parameters<typeof patchMethod>> = [body];
 
       if (this.isNamespaced) {
         args.push(this.getNamespace());
       }
 
       args.push(this.getName());
+      // @ts-expect-error
```

Comment:
What's this about? `ts-expect-error` is usually for testing invalid values.

If it's supposed to be ignored, can you please add a comment why?

Reply:
the way the arguments are passed into a variadic function makes it hard to make ts happy, this should just be ts-ignore, I'll change it